web/startScan/templates/startScan/vulnerabilities.html

```diff
@@ -79,10 +79,10 @@
 			{
 				"render": function ( data, type, row ) {
 					if(row['open_status']){
-						return `<div class="form-check mb-2 form-check-primary"><input type="checkbox" name="targets_checkbox['+ e + ']" class="float-start form-check-input" onchange="vuln_status_change(this, `+data+`);">\n<span class="new-control-indicator"></span><span style="visibility:hidden">c</span></div>`;
+						return `<div class="form-check mb-2 form-check-primary"><input type="checkbox" name="targets_checkbox['+ e + ']" class="float-start form-check-input vulnerability_checkbox" onchange="vuln_status_change(this, `+data+`);">\n<span class="new-control-indicator"></span><span style="visibility:hidden">c</span></div>`;
```

Comment:
You must move the onchange event to a onclick event on the Status badge  on the end of the line
![image](https://github.com/user-attachments/assets/b166b86f-9ad8-4853-bbab-d23da1f00c90)

Because when item is clicked, status is changed
![image](https://github.com/user-attachments/assets/2855f619-a3cd-47c6-a849-12a612208e64)

Reply:
Resolved, perfect :+1: